lib/match/runner.rb

```diff
@@ -90,5 +96,28 @@ def profile(params: nil, certificate_id: nil)
 
       return uuid
     end
+
+    def device_count_different?(params: nil, profile: nil)
```

Comment:
Instead of passing `params`, it would be nice to pass the actual values we need, in this case probably just the app identifier.